src/_posts/2017-01-09-added-apex-runner.md

```diff
@@ -24,7 +24,7 @@ Apex was built as stateful stream processor from the ground up. Operators [check
 
 ## Translation to Apex DAG
 
-A Beam runner needs to implement the translation from the Beam model to the underlying frameworks execution model. In the case of Apex, the runner will translate the pipeline into the [native (compositional, low level) DAG API](https://www.datatorrent.com/blog/tracing-dags-from-specification-to-execution/) (which is also the base for a number of other API that are available to specify applications that run on Apex). The DAG consists of operators (functional building blocks that are connected with streams. The runner provides the execution layer. In the case of Apex it is distributed stream processing, operators process data event by event. The minimum set of operators covers Beam’s primitive transforms: `ParDo.Bound`,  `ParDo.BoundMulti`, `Read.Unbounded`, `Read.Bounded`, `GroupByKey`, `Flatten.FlattenPCollectionList` etc.
+A Beam runner needs to implement the translation from the Beam model to the underlying frameworks execution model. In the case of Apex, the runner will translate the pipeline into the [native (compositional, low level) DAG API](https://www.datatorrent.com/blog/tracing-dags-from-specification-to-execution/) (which is also the base for a number of other API that are available to specify applications that run on Apex). The DAG consists of operators (functional building blocks that are connected with streams. The runner provides the execution layer. In the case of Apex it is distributed stream processing, operators process data event by event. The minimum set of operators covers Beam’s primitive transforms: `ParDo.SingleOutput`,  `ParDo.MultiOutput`, `Read.Unbounded`, `Read.Bounded`, `GroupByKey`, `Flatten.PCollections` etc.
```

Comment:
Please revert; we don't change blog posts -- they are dated and should be accurate as the date of publication.

If warranted, we can add a note at the top of the blog that it may be outdated -- but, I wouldn't change the text.